monkey/tests/monkey_island/cc/server_utils/test_island_logger.py

```diff
@@ -17,11 +12,13 @@ def mock_home_env(monkeypatch, tmpdir):
     monkeypatch.setenv("HOME", str(tmpdir))
 
 
-def test_expanduser_filename(mock_home_env, tmpdir, test_logger_config_path):
-    INFO_LOG = os.path.join(tmpdir, "info.log")
+def test_expanduser_filename(mock_home_env, tmpdir):
+    DATA_DIR = tmpdir
+    INFO_LOG = os.path.join(DATA_DIR, "monkey_island.log")
+    LOG_LEVEL = "DEBUG"
```

Comment:
Since this is testing the call to `logger.info()`, let's make this log level `INFO`. Or we could leave this as `DEBUG` and call `logger.debug()` instead.

We should also add another test where we set the log level to `INFO`, call `logger.debug()`, and verify that the file does not contain the line.